rs/nns/governance/src/governance.rs

```diff
@@ -5504,6 +5505,13 @@ impl Governance {
             ));
         }
 
+        let total_potential_voting_power = Some(total_potential_voting_power(
+            self.neuron_store.voting_eligible_neurons(now_seconds),
+            now_seconds,
+            &action,
+            ballots.len(),
+        )?);
+
```

Comment:
Figuring out what value to populate the new field with.

This field is not read yet; just written to. Reading will be done in the next PR (or maybe a couple PRs later?).